### ISSUE TITLE
Tags and `FormTokenField` disappear when user don't add comma or click Enter Key

### ISSUE THREAD

**Issue by phpbits** [WordPress/gutenberg#16788]
**Describe the bug**
`FormTokenField` bug on first value. Whenever you add value and do not add comma or press Enter Key the value disappears.

**To reproduce**
1. Create a post
2. Add only one tag and do not press Enter Key or add comma
3. Click the Paragraph block
4. Return to the Document inspector and the tag added is missing

**Expected behavior**
Value should be added automatically.

**Screenshots**
![Screen Capture on 2019-07-29 at 13-16-07](https://user-images.githubusercontent.com/3365507/62023352-18aabf00-b203-11e9-81c9-66e33c27edd1.gif)

**Comment by marekhrabe** [comment]
Thanks for reporting! I've made a PR #16849 with some further questions about this. The solution has some pros and cons we need to evaluate from usability and accessibility standpoints. I personally think this would be a valuable change

**Comment by joedolson** [comment]
As @afercia commented on PR #16849, the solution to this issue is to have a button to save tags. 

Any behavior we choose that happens on a change of focus or loss of visibility is potentially unpredictable. Right now, there are no instructions given to the user about how to save tags, only how to separate them (which also adds them, but isn't stated.)

A simple 'Add Tags' button would both communicate to the user that this is the mechanism to save a tag and provide them with the context that if they have *not* taken this action, the tags will not be saved.

**Comment by joedolson** [comment]
But the problem here is, in part, just that the value disappears, rather than retaining the userstate when the change of focus happens. We don't want the value saved to the database when the focus changes, but it should be retained for the user, so that they can save it later.

**Comment by ciampo** [comment]
@Mamaduka will https://github.com/WordPress/gutenberg/pull/80967 close / supersede this issue, too?

**Comment by Mamaduka** [comment]
Good find, @ciampo!

Yes, https://github.com/WordPress/gutenberg/pull/80967 will close/supersede this issue. The new component will use a "creatable" item for the same actions. No more accidental tag creations.

I think we can close this. Even with `FormTokenField`, we shouldn't create entries without the user's intent, and leaving the fields blank doesn't qualify as such, IMO.
